qemu/tests/nbd_unix_connection.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from virttest import error_context
+
+from avocado.utils import process
+
+from provider.nbd_image_export import QemuNBDExportImage
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Clone the system image1 with qemu-img
+    2) Export the cloned image with qemu-nbd(type=unix)
+    3) Stop to export nbd image
+    4) Repeat connect nbd image by nbdsh cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def run_nbd_connect_cmd(cmd):
+        result = process.run(cmd, timeout=5, ignore_status=True, shell=True)
+        if result.exit_status not in {0, -15}:
+            error_context.context("Command error:%s" % result.stderr.decode(),
+                                  logging.info)
+            logging.info('Stop export test image.')
+            nbd_export.stop_export()
```

Comment:
dup operation here for stop_export in finally block can handle it

Reply:
Thanks, checked, remove dup code.